tests/Microsoft.Experimental.Collections.Tests/Microsoft/Collections/Extensions/OrderedDictionaryTests.cs

```diff
@@ -0,0 +1,276 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Xunit;
+
+namespace Microsoft.Collections.Extensions.Tests
+{
+    public class OrderedDictionaryTests
+    {
+        [Fact]
+        public void TestBasicOrderedDictionaryOperations()
```

Comment:
nit: I would break these into separate API specific tests.

Reply:
I agree. Is there anywhere within the .NET Core repos with guidance on how to write good unit tests, especially in regards to generic collections.